contrib/fparser/fparser_ad.cc

```diff
@@ -576,7 +576,24 @@ Value_t FunctionParserADBase<Value_t>::Eval(const Value_t* Vars)
   if (compiledFunction == NULL)
     return FunctionParserBase<Value_t>::Eval(Vars);
   else
-    return (*compiledFunction)(Vars, pImmed, Epsilon<Value_t>::value);
+  {
+    // Use a local stack (alloca or new/delete), or the global stack for this parser object
```

Comment:
This block is more or less copied from the FParser Eval routine (@roystgnr, did you add this?)